### DISCUSSION TITLE
Looking for the name of some internal commands

### DISCUSSION THREAD

**Discussion by Archie-2021** [gokcehan/lf#1362] [Q&A] [answered]
Hey, 

I am looking for something similar to `gv` in vim. I don't know if there is sort of history for selected files or not. But I use `clear` and `unselect` at the end of most of my scripts but sometimes there is a need to do one thing on bunch of files then do something new with them. Not sure how I can select the same files again after running `unselect`

**Comment by joelim-work** [comment, marked as answer]
`lf` doesn't keep track of past selections, but you can manually write the selections to a file whenever it is updated and then restore them afterwards:

```shell
# update /tmp/selections.txt file whenever selection changes
map <space> :{{
    toggle
    down
    &echo "$fx" > /tmp/selections.txt
}}

# select files according to the contents of /tmp/selections.txt
map gv %{{
    lf -remote "send $id toggle $(xargs printf ' %q' < /tmp/selections.txt)"
}}
```

That should hopefully be a good starting point for you.

**Comment by Archie-2021** [comment]
Good idea, don't know why forget about such a solution. Very cool.
